src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,40 @@
 
 public class DefaultCountingOutRhymer {
 
-	private int[] numbers = new int[12];
+	private static final int EMPTY_STACK_VALUE = -1;
+	private static final int FULL_STACK_VALUE = 11;

```

Comment:
This constant is related to STACK_CAPACITY . Changing only one leads to an error (inconsistency).